mk/patches/patch-xmlrpc-xmlrpcCF.csproj.diff

```diff
@@ -28,26 +25,13 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 # SUCH DAMAGE.
 
-diff -r -u src/AssemblyBuildNumber.cs src/AssemblyBuildNumber.cs
```

Comment:
As discussed, please restore the patches for the build number because they keep track of our changes on top of the original library. In this case it should be 2.5.0.1 since it's the first time we have applied patches to the new library.